app/controllers/issues/states_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Issues::StatesController < AuthenticatedController
+  include ProjectScoped
+
+  def update_states
+    issues = current_project.issues.where(id: params[:ids])
+
+    if issues.update_all(state: state_params, updated_at: Time.now)
+      render json: { state: Issue.state_names[state_params] }
+    else
+      render json: { message: 'Something went wrong.' }
+    end
```

Comment:
Will the return of `issues.update_all` ever be false or nil?
If yes i think we should also handle on:error in the ajax call, if not we can just  do
```
issues.update_all(state: state_params, updated_at: Time.now)
render json: { state: Issue.state_names[state_params] }
```